.golangci.yml

```diff
@@ -122,6 +122,7 @@ issues:
         - typecheck
         - revive
         - unused
+        - gocyclo
```

Comment:
Dropped cyclo check from unit tests file to stop complaining when there are many tests (> 35) cases under a single suite